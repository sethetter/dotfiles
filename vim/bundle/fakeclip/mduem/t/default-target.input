#!/bin/bash

dir='t/,default-target'
makefile='t/,default-target.Makefile'
file_source='t/,default-target.source'
file_generated='t/,default-target.generated'

echo "$$" >"$file_source"
cat >"$makefile" <<END
TARGETS_GENERATED := $file_generated

.PHONY: first
first:
	echo 'first' >$file_generated

$file_generated: $file_source
	cp \$< \$@

include Makefile
END

make --always-make -f "$makefile" 2>&1 >/dev/null \
  "INSTALLATION_DIR=$dir" |
  sed 's/^Makefile:[0-9]*:/Makefile:%l:/'

diff "$file_source" "$file_generated" >/dev/null; echo "same $?"

rm -rf "$dir" "$makefile" "$file_source" "$file_generated"

# __END__
