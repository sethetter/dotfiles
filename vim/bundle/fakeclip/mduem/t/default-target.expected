Makefile:%l: .mduem/cache/Makefile.variables: No such file or directory
same 0
